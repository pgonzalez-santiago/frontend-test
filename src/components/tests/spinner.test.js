import React from 'react'
import Spinner from '../spinner'
import { shallow } from 'enzyme'

it('Spinner Dom structure has not change', () => {
  const wrapper = shallow(
    <Spinner
      color="#c94e50"
      size={50}
      style={''}/>
  )
  expect(wrapper).toMatchSnapshot()
})
