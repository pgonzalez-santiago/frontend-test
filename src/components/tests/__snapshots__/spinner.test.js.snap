// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Dom structure has not change 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Spinner
    color="#c94e50"
    size={50}
    style=""
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <onlyUpdateForKeys(Loader)
        className=""
        color="#c94e50"
        loading={true}
        size={50}
        sizeUnit="px"
      />,
      "className": "sweet-loading",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "className": "",
        "color": "#c94e50",
        "loading": true,
        "size": 50,
        "sizeUnit": "px",
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <onlyUpdateForKeys(Loader)
          className=""
          color="#c94e50"
          loading={true}
          size={50}
          sizeUnit="px"
        />,
        "className": "sweet-loading",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "className": "",
          "color": "#c94e50",
          "loading": true,
          "size": 50,
          "sizeUnit": "px",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
